semanticdb/scalac/library/src/main/scala-2.13/scala/meta/internal/semanticdb/scalac/CommandLineParser.scala

```diff
@@ -0,0 +1,5 @@
+package scala.meta.internal.semanticdb.scalac
+
+object CommandLineParser {
+  def tokenize(line: String): List[String] = scala.sys.process.Parser.tokenize(line)
```

Comment:
Why did this pop up only in 2.13.9? :thinking:

Reply:
In scala/scala this class was deleted and replaced by sys.process one after 2.13.8 - https://github.com/scala/scala/pull/10057